servicetalk-opentracing-zipkin-publisher/src/main/java/io/servicetalk/opentracing/zipkin/publisher/ZipkinPublisher.java

```diff
@@ -178,6 +178,12 @@ public void onSpanFinished(final InMemorySpan span, long durationMicros) {
             builder.kind(Span.Kind.SERVER);
         } else if (Tags.SPAN_KIND_CLIENT.equals(type)) {
             builder.kind(Span.Kind.CLIENT);
+        } else if (Tags.SPAN_KIND_PRODUCER.equals(type)) {
+            builder.kind(Span.Kind.PRODUCER);
+        } else if (Tags.SPAN_KIND_CONSUMER.equals(type)) {
+            builder.kind(Span.Kind.CONSUMER);
+        } else {
+            LOGGER.warn("Found unsupported kind '{}' in span tags. Not setting span.kind", type);
```

Comment:
Lets skip the warning here. Span kind is not a required field so no-value is also valid.

Reply:
Please consider updating the copyright of the file 2018 -> 2018, 2021